### PR TITLE
Change DPSG url

### PR DESCRIPTION
**Pull request by hobadams**
- change DPSG URL 

- change iDeal to iDEAL

### CODE REVIEW

File: dpsg/docs/integrations/adyen/config.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ From their you can set up `Standard Notification` webhook.
 ### Webhook Config
 
 **Transport**
-- URL: https://dpsg.dev.deity.cloud/api/payment/webhook/{organization}/{project}/{environment}/adyen (replacing `{organization}`, `{project}` and `{environment}` with your details).
+- URL: https://dpsg.deity.cloud/api/payment/webhook/{organization}/{project}/{environment}/adyen (replacing `{organization}`, `{project}` and `{environment}` with your details).
```
Comment (jonas-at-deity) [inline dpsg/docs/integrations/adyen/config.md:57]: We have multiple potential regions, so this should be:
`https://dpsg.{region}.deity.cloud/api/payment/webhook/{organization}/{project}/{environment}/adyen`

And then mention that if it is `eu`, the region can be omitted.

Comment (hobadams) [reply dpsg/docs/integrations/adyen/config.md]: @jonas-at-deity - this is ready for re-review

File: dpsg/docs/integrations/adyen/config.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ From their you can set up `Standard Notification` webhook.
 ### Webhook Config
 
 **Transport**
-- URL: https://dpsg.`{region}`.deity.cloud/api/payment/webhook/{organization}/{project}/{environment}/adyen (replacing `{organization}`, `{project}` and `{environment}` with your details). If your region is Europe is will be omitted (https://dpsg.deity.cloud).
+- URL: https://dpsg.{region}.deity.cloud/api/payment/webhook/{organization}/{project}/{environment}/adyen (replacing `{organization}`, `{project}` and `{environment}` with your details). If your region is Europe is will be omitted (https://dpsg.deity.cloud).
```
Comment (jonas-at-deity) [inline dpsg/docs/integrations/adyen/config.md:57]: Typo and it should not be `will` but `should`

`If your region is Europe, the region should be omitted from the url (https://dpsg.deity.cloud).`

Comment (hobadams) [reply dpsg/docs/integrations/adyen/config.md]: @jonas-at-deity - i'm imagining customers getting the URL from the env config, as in, they will already have the URL. so  in that sense `will` makes sense.  I'm happy either way though if you feel strongly about this?

**Review by jonas-at-deity — APPROVED** [review]

**Pull request merged**